bgpd/bgp_vty.c

```diff
@@ -6477,10 +6418,18 @@ DEFUN (vpn_policy_rt,
 
 			vpn_leak_prechange(dir, afi, bgp_get_default(), bgp);
```

Comment:
I don't see a check on whether import or export is enabled for this VRF or not. Without that, the pre and post change operations may walk routing tables before they bail out. Or maybe it is there some place that I missed? This comment may apply to other handlers too.

Reply:
Added checks.